api/src/Controller/NotificationController.php

```diff
@@ -96,13 +98,23 @@ public function  createOrEditEmployeeAction(Request $request, CommonGroundServic
             $employeeService = new EmployeeService($commonGroundService);
             $employee = $employeeService->checkOrganization($employee);
             // Create/update a user for it in the gateway with correct user groups
-            $userService = new UserService($commonGroundService);
+            $mailService = new MailService($commonGroundService, $twig, $parameterBag);
+            $userService = new UserService($commonGroundService, $mailService);
             $user = $userService->saveEmployeeUser($employee, $data['action']);
+            if (isset($user['employee'])) {
+                $employee = $user['employee'];
+                unset($user['employee']);
+            }
+            if (isset($user['message'])) {
+                $message = $user['message'];
+                unset($user['message']);
+            }
```

Comment:
Wat doet dit precies?

Reply:
ah, dat is alleen maar voor de response die de logic service zelf terug geeft, niet super belangrijk tenzij je online zou testen of logs gaat bekijken misschien.
Het is een wat minder nette manier om gegevens vanuit mijn functie om te zetten naar een iets nettere opbouw voor de response.